### PR TITLE
Format log

### PR DESCRIPTION
**Pull request by nejisama**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Sign the CLA
Make sure you have signed the [CLA](https://www.clahub.com/agreements/alipay/sofa-mosn)

### Solutions

1. Add function Alertf in DefaultLogger and ProxyLogger
2. change some log calls from Errorf to Alertf

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases.
And you need to show the ut result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: pkg/log/errorlog.go [hunk 1]
```diff
@@ -46,9 +48,16 @@ func CreateDefaultErrorLogger(output string, level Level) (ErrorLogger, error) {
 }
 
 // default logger format:
-// {time} [{level}] {content}
+// {time} [{level}] [{error code}] {content}
+// default error code is normal
+const defaultErrorCode = "normal"
+
 func (l *errorLogger) formatter(lvPre string, format string) string {
-	return logTime() + " " + lvPre + " " + format
```
Comment (neverhook) [inline pkg/log/errorlog.go]: 这里感觉有问题，预期占位符是给 level = ERROR 且没有调用 alert 传递 error code 的场景使用的，目前直接替换 formatter 的话会导致 level = INFO、DEBUG 也会打印占位符
